### DISCUSSION TITLE
Error Message

### DISCUSSION THREAD

**Discussion by SCFrostino** [TomSchimansky/CustomTkinter#1292] [Q&A] [answered]
Is there a way to add an error message, I can get an error message to work using regular tkinter however I want the error message to be more modern like customtkinter

![image](https://user-images.githubusercontent.com/91733542/222002521-9b0aebd9-89a9-41c4-8621-8012cb5ed702.png)

**Comment by ElectricCandlelight** [comment, marked as answer]
Could try #1279 

[https://github.com/Akascape/CTkMessagebox](https://github.com/Akascape/CTkMessagebox)

**Reply by SCFrostino** [reply]
Perfect, this is exactly what I am looking for, will try this out soon thank you

**Reply by SCFrostino** [reply]
Just a quick question? im not sure how experienced you are in using this but I just tried it out and noticed that there is no border on the window compared to a customtkinter window. Are you aware if there are any ways to change this?

**Reply by Akascape** [reply]
@SCFrostino The borders are removed using the `overrideredirect` method in order to match the UI. I can add a small border parameter around the frame if you wish.
Let me know if you are talking about the overall border or the window title bar/header?

**Reply by SCFrostino** [reply]
Hi, both, i mean around the whole border and maybe making the top bar a little more prominent. Sorry for the glare in the image
![image](https://user-images.githubusercontent.com/91733542/222445585-837cfedf-6fff-4477-beeb-36e6b5d14b6d.jpg)

**Reply by Akascape** [reply]
@SCFrostino Here is a tweak you can implement if you want the basic header:
**(Make sure to update ctkmessagebox)**
```python
msg = CTkMessagebox(message="Your message", icon="cancel", header=True, height=150, border_width=0)
```

**Reply by Akascape** [reply]
@SCFrostino I don't think it will look that good if you add the header back. Instead, you can **remove the red button** by adding `cancel_button_color='transparent'` and use `border_width=2`, it will look cleaner.

![Screenshot](https://user-images.githubusercontent.com/89206401/222474648-f543105d-5cfb-437d-bb3e-e804d682b57a.jpg)
